README.md

```diff
@@ -7,8 +7,13 @@ this process.
 
 It is compatible with Fortinet VPNs.
 
-
+Usage
```

Comment:
This section name is not standard, but it's a brilliant idea.

Reply:
To be clear, are you fine with this addition?

Reply:
Yes!